### ISSUE TITLE
Tastendruck kann zu Crash führen

### ISSUE THREAD

**Issue by Jarani85** [AlternateLife/Bugs#191]
### Beschreibe den Fehler
Der Arm des Chars ist ausgestreckt ("B" oder "Ö" Funktion). Man drückt "F" oder "G" und das Spiel crashed ohne jegliche Fehlermeldung. Es Freezed einfach ein und das Windows Fenster, dass die Anwendung nicht mehr reagiert, erscheint.

Ist mir bisher jeweils nur bei diesen beiden Tasten aufgefallen und hatte es heute eben zwei Mal.

### Reproduktion des Fehlers
Schwer zu reproduzieren, tritt willkürlich auf bei ausgestreckten Armen

- Arm ausstrecken
- "F" oder "G" drücken
- Spiel friert ein und stürzt ab

### Erwartetes Verhalten
Kein Freeze und kein Spielabsturz

### Name der/des beteiligten Charakter(e)
Ivory Stokes

### System
- Betriebssystem: Windows 10
- Browser: Chrome
- GTA Version: socialclub

**Comment by Cryma** [comment]
Wenn das Spiel einfriert, erscheint dort eine Meldung von RageMP im Hintergrund?

**Comment by Jarani85** [comment]
Nein, das Spiel friert ein und es kommt die Windows-Meldung, dass die Anwendung nicht reagiert. Keine Meldung von RageMP
